Objects/typevarobject.c

```diff
@@ -443,45 +443,42 @@ static PyMethodDef typevar_methods[] = {
 PyDoc_STRVAR(typevar_doc,
 "Type variable.\n\
 \n\
-Usage::\n\
-\n\
-  T = TypeVar('T')  # Can be anything\n\
-  A = TypeVar('A', str, bytes)  # Must be str or bytes\n\
+The preferred way to construct a type variable is via the dedicated syntax\n\
+for :ref:`generic functions <generic-functions>`,\n\
+:ref:`generic classes <generic-classes>`, and\n\
+:ref:`generic type aliases <generic-type-aliases>`::\n\
```

Comment:
The Sphinx markup doesn't do anything if you're calling `help(TypeVar)` in the REPL, and makes it slightly difficult to read:

```suggestion
The preferred way to construct a type variable is via the dedicated syntax\n\
for generic functions, generic classes and generic type aliases::\n\
```